installer/uninstaller.sh

```diff
@@ -15,12 +15,20 @@
 # limitations under the License.
 #
 
-charts_list="management-api-chart crd-subchart openldap dex-subchart ingress"
-HELM_OUTPUT=`helm ls --output json`
-HELM_LIST=`jq --arg namearg "Releases" '.[$namearg]' <<< $HELM_OUTPUT`
+delete_release_if_included_in_list() {
+    chart_name=$1
+    release_name=$2
+    [[ $CHARTS_LIST =~ (^|[[:space:]])$chart_name($|[[:space:]]) ]] && echo "Release will be deleted" && helm del --debug --purge $release_name || echo 'Release wont be deleted'
+}
+
+CHARTS_LIST="management-api-chart crd-subchart openldap dex-subchart ingress minio"
```

Comment:
it would be nice to have `imm` prefixed charts to e sure that we are remove only our `imm-ingess`